src/app/clusters/fan-control-server/fan-control-server.cpp

```diff
@@ -298,6 +298,17 @@ MatterFanControlClusterServerPreAttributeChangedCallback(const ConcreteAttribute
         }
         break;
     }
+    case PercentCurrent::Id: {
+        res = Status::Success;
+        // ReportRequiredByWhatever - Something that get's triggered on edges to force reporting
+        if (ReportRequiredByWhatever) {
```

Comment:
OK, but where do we store the state to compute this value?  And at that point, it seems like it should also just store the last-reported PercentCurrent value and cluster users should call a cluster API to set the attribute instead of directly poking the attribute store....

Reply:
Right, that would be the next step.  I had similar thoughts about local storage.  Wanted to see if the overall approach (enforce in the PreAttributeChangeCallback with support from the IM for the actual reporting or not) seemed sane.